src/lualdap.c

```diff
@@ -909,7 +909,7 @@ static int lualdap_search_tostring (lua_State *L) {
 ** Create a metatable.
 */
 static int lualdap_createmeta (lua_State *L) {
-	const luaL_Reg methods[] = {
+	static const luaL_Reg connection_methods[] = {
```

Comment:
Both static and name change are good. Since the array is now static, it could go directly into the file scope, instead of into the function.

Reply:
I generally prefer it inside the function unless it is reused elsewhere.
Reducing scope is good for catching unwanted interactions.